include/global.h

```diff
@@ -97,6 +97,7 @@
 #define T2_READ_16(ptr) ((ptr)[0] + ((ptr)[1] << 8))
 #define T2_READ_32(ptr) ((ptr)[0] + ((ptr)[1] << 8) + ((ptr)[2] << 16) + ((ptr)[3] << 24))
 #define T2_READ_PTR(ptr) (void*) T2_READ_32(ptr)
+#define T2_READ_U8_PTR(ptr) (u8*) T2_READ_32(ptr)
```

Comment:
Not necessary, just cast the result of the preceding macro to `(u8 *)` in source